Parser/pegen/pegen.c

```diff
@@ -860,7 +860,7 @@ interactive_exit(Parser *p)
 
 /* Creates a single-element asdl_seq* that contains a */
 asdl_seq *
-singleton_seq(Parser *p, void *a)
+_PyPegen_singleton_seq(Parser *p, void *a)
```

Comment:
Maybe we can move the various sequence helpers into Python/asdl.c. Those functions have a `_Py_asdl_` prefix. Though those APIs take a `PyArena *` argument, so we'd have to pass `p->arena` instead of `p`. And the convention is that the arena is the last arg, not the first. So probably for a separate PR?